pkg/schedule/schedulers/hot_region_v2.go

```diff
@@ -93,6 +132,7 @@ func (bs *balanceSolver) pickCheckPolicyV2() {
 	}
 }
 
+// filterUniformStoreV2 filters stores by stddev.
```

Comment:
this term `Stddev` is confused me, can you give more info?

Reply:
added